_projects/extend_linter_integration.md

```diff
@@ -91,9 +92,13 @@ difficult to debug things like:
 >           raise AssertionError(str(err) + " \n" + "\n".join(msg))
 E           AssertionError: 0 == 0 : The local bear 'CheckstyleBear' yields no result although it should.
 E           Running bear CheckstyleBear...
-E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/.local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/CheckstyleGood.java'
+E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/
+			.local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
+			CheckstyleGood.java'
 E           Running bear CheckstyleBear...
-E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/.local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/CheckstyleBad.java'
+E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/
```

Comment:
break it after `.jar` so that the `-c` is together with its argument on the next line, and dont break file paths across multiple lines